packages/fxa-auth-client/lib/client.ts

```diff
@@ -1284,16 +1284,16 @@ export default class AuthClient {
     );
   }
 
-  // TODO: Update to POST in FXA-7400
-  async getRecoveryKeyHint(
-    sessionToken: hexstring | undefined,
-    email?: string
-  ): Promise<{ hint: string | null }> {
-    if (sessionToken) {
-      return this.sessionGet('/recoveryKey/hint', sessionToken);
-    }
-    return this.request('GET', `/recoveryKey/hint?email=${email}`);
-  }
+  // TODO: Review in FXA-7400 - possibly convert to POST to pass payload instead of using param, and enforce rate limiting
```

Comment:
Yes, payload is the email.

I was thinking GET is right for this too since we're only receiving information, but someone (I think Vijay?) had suggested we switch to POST a few months ago because HAPI doesn't allow passing the email in body (only in params). Either way, the important change to make in the follow-up ticket is to ensure that this call is rate-limited and has an auth strategy based on the initiated password reset/email confirmation (or session token if called from /settings) - I'll update the follow-up ticket details.

Reply:
One point in favor of POST is that we've had emails exposed in GET strings before (somehow -- they get into logs or crawlers or add-ons or whatever) and they can float around on the internet that way.  Something we should discuss when we implement the final version of this.